### PR TITLE
feat: Support WASM external transformers using Wazero runtime

### PR DESCRIPTION
**Pull request by shashank381**

### CODE REVIEW

**Comment by kmehant** [comment]
@shashank381 Thank you, one of us would review it.

File: transformer/external/wasmtransformer.go [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+	for i := range preopens {
+		fsconfig = wazero.NewFSConfig().WithDirMount(preopens[i][0], preopens[i][1])
+	}
+
+	config := wazero.NewModuleConfig().
+		WithStdout(os.Stdout).WithStderr(os.Stderr).WithFSConfig(fsconfig)
+
+	envVars := t.prepareEnv()
+	for _, envVar := range envVars {
+		keyValue := strings.SplitN(envVar, wasmEnvDelimiter, 2)
```
Comment (HarikrishnanBalagopal) [inline transformer/external/wasmtransformer.go:271]: Why are we joining the key and value with = delimiter and then splitting again?

**Review by HarikrishnanBalagopal — CHANGES_REQUESTED** [review]

File: transformer/external/wasmtransformer.go (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply transformer/external/wasmtransformer.go:271]: Also fix the DCO check by signing

**Comment by kmehant** [comment]
```
INFO[0001] Planning started on its sub directories      
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/django/pom.xml: file does not exist

wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/django/simplewebapp/pom.xml: file does not exist

wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/django/webroot/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/django/webroot/migrations/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/golang/pom.xml: file does not exist

wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/gradle/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/gradle/wrapper/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/main/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/main/java/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/main/java/com/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/main/java/com/myuser/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/main/webapp/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-gradle/src/main/webapp/WEB-INF/pom.xml: file does not exist
INFO[0004] Identified 1 named services and 0 to-be-named services in java-maven 
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/java-war/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/nodejs/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/php/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/python/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/ruby/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/ruby/views/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/rust/pom.xml: file does not exist
wasm stat failed: stat /Users/haribala/temp/temp-m2k-cli-wasm/t1/language-platforms/rust/src/pom.xml: file does not exist
INFO[0005] Planning finished on its sub directories    
```

Can we check stat is being checked in every possible location in the source artifact folders? @shashank381

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

**Pull request merged**